bin/pulsar-perf

```diff
@@ -93,7 +93,7 @@ or command is the full name of a class with a defined main() method.
 
 Environment variables:
    PULSAR_LOG_CONF               Log4j configuration file (default $DEFAULT_LOG_CONF)
-   PULSAR_CLIENT_CONF          Configuration file for client (default: $DEFAULT_CLIENT_CONF)
+   PULSAR_CLIENT_CONF            Configuration file for client (default: $DEFAULT_CLIENT_CONF)
```

Comment:
The indentation here is also fix by the way.
[help](https://github.com/apache/pulsar/blob/4aef300a2d50cc1e79622fc3120bb1e7a8f8335e/bin/pulsar-perf#L90)

Reply:
Changed, thanks!

Reply:
> Changed, thanks!

well done.